## 50_edit-env_autoload.diff by Peter S Galbraith <psg@debian.org>

--- a/elisp/emacs-goodies-el/edit-env.el	2003-11-12 20:17:21.000000000 -0500
+++ b/elisp/emacs-goodies-el/edit-env.el	2003-11-14 15:00:01.000000000 -0500
@@ -111,6 +111,7 @@
 	       (list (widget-get widget 'environment-variable-name)
 		     widget)))
   
+;;;###autoload
 (defun edit-env ()
   "Display, edit, delete and add environment variables."
   (interactive)
